canonicalwebteam/blog/flaskblog/views.py

```diff
@@ -1,10 +1,10 @@
-import flask
-
 from canonicalwebteam.blog import wordpress_api as api
 from canonicalwebteam.blog import logic
 
 
 def build_blueprint(blog_title, tags_id, tag_name):
+    import flask
```

Comment:
is there a reason why you added this here?

Reply:
Have updated it. The dynamic import now happens in the main `app.py`. I think that way is cleaner.

The original reason was, to not having to install flask in a django app